helm-charts/support/values.yaml

```diff
@@ -100,6 +100,26 @@ prometheus:
       - key: k8s.dask.org/dedicated
         value: worker
         effect: NoSchedule
+    # resources for the node-exporter was set after inspecting cpu and memory
+    # use in a few clusters using prometheus queries from
```

Comment:
Could you maybe add a note that upstream don't set deliberately leave setting this to user choice as per

https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/values.yaml#L53

Reply:
If you do `helm create test` and and `cat test/values.yaml` you will find this comment - its a boilerplate comment from `helm create` rather than the prometheus community maintaining that chart.

Our support chart isn't packaged and distributed for others to install, its used as our bundled configuration of various charts we want to install with misc common configuration.

Do you have a concrete suggestion of a comment that you think should be added?

---

This is from `helm create test`'s values.yaml file:

```yaml
resources: {}
  # We usually recommend not to specify default resources and to leave this as a conscious
  # choice for the user. This also increases chances charts run on environments with little
  # resources, such as Minikube. If you do want to specify resources, uncomment the following
  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
  # limits:
  #   cpu: 100m
  #   memory: 128Mi
  # requests:
  #   cpu: 100m
  #   memory: 128Mi
```

Reply:
Ok. Just ignore this request